docs/release-process.md

```diff
@@ -150,13 +150,23 @@ and create a Pull Request on GitHub.
 
 #### 6. Publish the release candidate to the npm
 
-Set the npm version for the release candidate
+If you don't have permission to publish `pulsar-client` to the npm registry, ask other committers to grant that permission.
+
 ```sh
-npm version 1.8.0-rc.1 --no-git-tag-version
-```
+# Set the npm version for the release candidate
+npm version 1.x.0-rc.1 --no-git-tag-version
+
+# Create or verify a user account in the npm registry
+$ npm adduser
+
+Username: foobar
+Password: ********
+Email: (this IS public) foobar@apache.org
 
-You can use `npm pack` and test the package before publishing. 
-If there are no problems, then use `npm publish` to publish to the npm.
+# Publish the rc version package to the npm registry
+# We use the `rc` tag to distinguish the release candidate from the official release.
+$ npm publish --tag rc
```

Comment:
If tag is set, this version will not be latest. 

https://docs.npmjs.com/cli/v9/commands/npm-dist-tag#description
```
Publishing a package sets the latest tag to the published version unless the --tag option is used. For example, npm publish --tag=beta.

By default, npm install <pkg> (without any @<version> or @<tag> specifier) installs the latest tag.
```